CHANGELOG

```diff
@@ -1,6 +1,9 @@
 v3.9.7 (XXXX-XX-XX)
 -------------------
 
+* Prevent agency configuration confusion by an agent which comes back without
```

Comment:
Nit: 3.9.7 has already been tagged. Potentially this change will target 3.9.8. But no need to change it now.